voxcell/voxel_data.py

```diff
@@ -140,6 +140,7 @@ def save_nrrd(self, nrrd_path, encoding=None):
             # before specifying the volume 3D axes.
             # For instance, a volume of orientations (3D direction vectors or quaternions)
             # or of RGB colors (3D int vectors) requires an initial None value.
+            nrrd.SPACE_DIRECTIONS_TYPE = 'double vector list'
             space_directions = [None] * dim_defect + list(space_directions)
```

Comment:
```suggestion
            space_directions = [[None] * self.ndim] * dim_defect + list(space_directions)
```
Could you give this a spin? It should fix the issue with the existing default configuration as suggested in https://github.com/mhe/pynrrd/issues/162

Reply:
whatever is your prefered solution, they are equivalent

Reply:
Changing globals may result in unexpected behavior, especially when pynrrd is used alongside voxcell for some reason and the latter changes the global state when it is not expected by the first. Highly unlikely, yet not impossible.

If changing the global is the way to go, I would suggest to wrap the `nrrd.read` in a context manager that changes the global only for the reading operation and then returns it to the previous value. It is unfortunate that we have to deal with global state but that would be the cleanest approach imo.

For this issue, it would be sufficient to just change to what the default is expecting and I could open a PR with the context manager to make the reading stable regardless of what defaults pyrrnd has in place or plans to change in the future.

@mgeplf wdyt?

Reply:
Note that changing a global would not be thread safe even with a context manager, unless some lock mechanism is used (in case we care about thread safety).
I would prefer to avoid modifying globals if possible, using something like `[None] * self.ndim]` seems safer.